src/pages/EditReportFieldDropdownPage.tsx

```diff
@@ -63,8 +77,8 @@ function EditReportFieldDropdownPage({fieldName, onSubmit, fieldID, fieldValue,
             },
             {
                 title: translate('common.all'),
-                shouldShow: filteredRestOfOptions.length > 0,
-                data: filteredRestOfOptions.map((option) => ({
+                shouldShow: restOfOptions.length > 0,
+                data: restOfOptions.map((option) => ({
```

Comment:
Under `all' we should display everything.
Please check behavior for categories and tags

<img width="370" alt="Screenshot 2024-03-04 at 11 23 24 AM" src="https://github.com/Expensify/App/assets/32012005/adde4f54-c30c-4d67-87a8-d4d8733bd4c9">

Reply:
Hmm, if we show it on both category, there will be 2 checkmarks.

<img width="378" alt="image" src="https://github.com/Expensify/App/assets/50919443/5193e746-ac6d-4d54-9691-730d6ab472e1">

For category, we have a selected section and it's not included in either Recent or All.

Reply:
@bernhardoj I think we should filter the selected option like we did with category.
https://github.com/Expensify/App/blob/8f58c576dfcd7944d26186302cc6b4319782a1b3/src/libs/OptionsListUtils.ts#L1045

Reply:
This is how it will looks if we follow category page.

<img width="381" alt="image" src="https://github.com/Expensify/App/assets/50919443/df76df2d-34c5-4c9a-a3a7-ad693385debc">

Reply:
Yes, "All" section includes all options except selected

<img width="373" alt="Screenshot 2024-03-04 at 2 31 25 PM" src="https://github.com/Expensify/App/assets/32012005/2cbc815d-1869-489e-81d8-c215beeb9395">

Reply:
Updated